src/demo/button/button.stories.mdx

```diff
@@ -1,12 +1,11 @@
 import { Story, Preview, Meta } from '@storybook/addon-docs/blocks';
 import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
-import { withA11y } from '@storybook/addon-a11y';
 import template from './button.twig';
 import './button.scss';
 
 <Meta
   title="Demo/Button"
-  decorators={[withKnobs, withA11y]}/>
+  decorators={[withKnobs]}/>
```

Comment:
What did this change do?

Reply:
Still trying to understand how this is working. Is it no longer necessary to specify withA11y because it is in the preview.js file?

Reply:
The way the accessibility add-on works, it applies to _all_ stories once it's been added anywhere. Now that I've added a `preview.js` config file, I can define it _there_ once, which felt like a more intuitive place to initialize an add-on that applies everywhere.